linter/whitelists/default.txt

```diff
@@ -137,6 +138,7 @@ Pixabay
 PNG
 Podcasts
 POIs
+Protocol
```

Comment:
Are you sure this is suitable for the default whitelist? You only provided the German translation.